charts/datree-admission-webhook/templates/serviceaccount.yaml

```diff
@@ -36,9 +36,11 @@ metadata:
   name: datree-cleanup-ping-hook-pre-delete
   labels: {{include "datree.labels" . | nindent 4}}
   namespace: {{template "datree.namespace" .}}
-  {{- with .Values.customAnnotations }}
-  annotations: {{ toYaml . | nindent 4 }}
-  {{- end }}
+  annotations:
+    argocd.argoproj.io/hook: Skip
```

Comment:
because the post delete hook is skipped for argocd we don't need the service account